cli/src/main/package/bin/ide.bat

```diff
@@ -5,6 +5,22 @@ Set _fBGreen=[92m
 Set _fBRed=[91m
 Set _RESET=[0m
 
+rem Add Git PATH entries for CMD - https://github.com/devonfw/IDEasy/issues/764
+for %%H in ( HKEY_LOCAL_MACHINE HKEY_CURRENT_USER ) do for /F "usebackq tokens=2*" %%O in (`call "%SystemRoot%"\system32\reg.exe query "%%H\Software\GitForWindows" /v "InstallPath" 2^>nul ^| "%SystemRoot%\system32\findstr.exe" REG_SZ`) do set GIT_HOME=%%P
+
+set "GIT_BIN=%GIT_HOME%\usr\bin"
+set "GIT_CORE=%GIT_HOME%\mingw64\libexec\git-core"
+
+if exist "%GIT_HOME%\bin\bash.exe" (
+	echo "%PATH%" | find /i "%GIT_BIN%">nul  || set PATH=%PATH%;%GIT_BIN%
+)
+
+if exist "%GIT_HOME%\bin\bash.exe" (
+	echo "%PATH%" | find /i "%GIT_CORE%">nul  || set PATH=%PATH%;%GIT_CORE%
+	REM KEEP the whitespace after call to clear ERRORLEVEL back to 0 otherwise it's 1 when PATH entries not found in find
+	call
+)
```

Comment:
For clarification... the `call_ ` is there to set the errorlevel to 0 since the `find /i "%GIT_CORE%">nul` is neq 0 when the entry is not found in the PATH and we get `Error: IDEasy failed with exit code %ERRORLEVEL%`